### DISCUSSION TITLE
http: TLS handshake timeout, Attempt 1

### DISCUSSION THREAD

**Discussion by 52znn** [AlistGo/alist#1421] [Q&A]
![image](https://user-images.githubusercontent.com/42986378/181675072-bfbe8ba0-3ae3-4ec9-9b82-d59be1f11489.png)
请问这个超时是什么原因呢，导致alist目录加载一直转圈，很慢才加载出来。
服务器是韩国的

**Comment by gengjiawen** [comment]
+1, 请问你后面怎么解决的？

**Reply by MichaelChao02** [reply]
我之前挂载115的时候遇到`TLS handshake timeout`,后来网上查了原因是因为开了ipv6. 关掉ipv6以后也确实好了，但很奇怪的是不关ipv6的话这个错误有时候有有时候没有。

**Reply by gengjiawen** [reply]
> 我之前挂载115的时候遇到`TLS handshake timeout`,后来网上查了原因是因为开了ipv6. 关掉ipv6以后也确实好了，但很奇怪的是不关ipv6的话这个错误有时候有有时候没有。

请教下你怎么关的，整个机器关掉IPv6 的吗
